scripts/check-locales.sh

```diff
@@ -0,0 +1,3 @@
+#! /bin/bash
+
+cmp <(jq 'path(..)|[.[]|tostring]|join(".")' src/locales/en.json) <(jq 'path(..)|[[.[]| select(length > 0)]  |tostring]|join(".")' src/locales/fr.json)
```

Comment:
Nice !

`[.[]|tostring]` can be written `map(tostring)` :)
`[.[]| select(length > 0)]` can be written `map(select(length > 0))`